cfme/tests/infrastructure/test_providers.py

```diff
@@ -334,6 +336,53 @@ def test_provider_rhv_create_delete_tls(request, provider, verify_tls):
     prov.wait_for_delete()
 
 
+@pytest.mark.rhv3
+@test_requirements.rhev
+@pytest.mark.meta(automates=[1691109, 1731237])
+@pytest.mark.provider([RHEVMProvider], selector=ONE_PER_VERSION, override=True, scope="function")
+def test_rhv_guest_devices_count(appliance, has_no_providers, provider, request):
+    """
+    Polarion:
+        assignee: anikifor
+        casecomponent: Infra
+        caseimportance: high
```

Comment:
If no `caseimportance` given it will treated as `high`

Reply:
Thanks, I know this but I prefer to be explicit if possible :)

Reply:
Okay. It's your call :smile: